package-lock.json

```diff
@@ -3928,6 +4014,11 @@
         "camelcase": "^5.0.0",
         "decamelize": "^1.2.0"
       }
+    },
+    "yarn": {
```

Comment:
Any reason for including *yarn*?

Reply:
Sadly it's part of `@oclif/plugins`. It's overkill in my opinion, but it's upstream.